DOTNET-example/README.md

```diff
@@ -1,18 +1,17 @@
 # Instrumenting .NET Apps with OpenTelemetry
 
-## Auto-Instrumentation
+## Instrument Using a Library
```

Comment:
@oppegard : Changed this as discussed at the meeting.

Reply:
Hi Travis,
Thank you! Let's add the auto instrumentation steps later on as discussed at the meeting. Will leave this content as we discussed.